src/db/tables/emailAddresses.ts

```diff
@@ -326,39 +306,6 @@ async function getUserEmails(userId: number): Promise<EmailAddressRow[]> {
 }
 /* c8 ignore stop */
 
-// This is used by SES callbacks to remove email addresses when recipients
-// perma-bounce or mark our emails as spam
-// Removes from either subscribers or email_addresses as necessary
-// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
-/* c8 ignore start */
-async function removeEmail(email: string) {
```

Comment:
I'll just note that this function and the comment sounded like it might still be used, but I tracked it down to getting added in 649d27a8ce94041ee3d5ea71707bfa6bf66f6a05 to support `controllers/ses.js`, which is long gone, so I think we're good there.

Reply:
This does sound like something we should be implementing... however we do keep an eye on the bounce rate and it hasn't been a problem, so I wonder if this is actually necessary or not. It's probably worth investigating as a followup ticket.